x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_actions/results.js

```diff
@@ -37,29 +44,59 @@ export function ResultLinks({ jobs }) {
   const singleMetricEnabled = jobs.length === 1 && jobs[0].isSingleMetricViewerJob;
   const jobActionsDisabled = jobs.length === 1 && jobs[0].deleting === true;
   const { createLinkWithUserDefaults } = useCreateADLinks();
+  const timeSeriesExplorerLink = useMemo(
+    () => createLinkWithUserDefaults('timeseriesexplorer', jobs),
+    [jobs]
+  );
+  const anomalyExplorerLink = useMemo(() => createLinkWithUserDefaults('explorer', jobs), [jobs]);
+
   return (
     <React.Fragment>
       {singleMetricVisible && (
         <EuiToolTip position="bottom" content={openJobsInSingleMetricViewerText}>
+          {isManagementTable ? (
+            <EuiButtonIcon
+              href={`${basePath.get()}/app/ml/${timeSeriesExplorerLink}`}
+              iconType="visLine"
+              aria-label={openJobsInSingleMetricViewerText}
+              className="results-button"
+              isDisabled={singleMetricEnabled === false || jobActionsDisabled === true}
+              data-test-subj="mlOpenJobsInSingleMetricViewerFromManagementButton"
+            />
+          ) : (
+            <Link to={timeSeriesExplorerLink}>
+              <EuiButtonIcon
+                iconType="visLine"
+                aria-label={openJobsInSingleMetricViewerText}
+                className="results-button"
+                isDisabled={singleMetricEnabled === false || jobActionsDisabled === true}
+                data-test-subj="mlOpenJobsInSingleMetricViewerButton"
+              />
+            </Link>
+          )}
+        </EuiToolTip>
+      )}
+      <EuiToolTip position="bottom" content={openJobsInAnomalyExplorerText}>
+        {isManagementTable ? (
           <EuiButtonIcon
-            href={createLinkWithUserDefaults('timeseriesexplorer', jobs)}
-            iconType="visLine"
+            href={`${basePath.get()}/app/ml/${anomalyExplorerLink}`}
```

Comment:
Link should be created by ML URL generator

Reply:
Same as above